src/main/webapp/js/stats-page-loader.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Fetch stats and display them in the page.
+ * @return {JSON} stats
```

Comment:
It looks like this function doesn't return anything 🙂I would just update the comment to remove the @return.